README.md

```diff
@@ -0,0 +1,55 @@
+# What is Weechat-Matrix?
+
+[Weechat](https://weechat.org/) is an extensible chat client.
+
+[Matrix](https://matrix.org/blog/home) is an open network for secure, decentralized communication.
+
+[Weechat-Matrix](https://github.com/poljar/weechat-matrix/) is a Python plugin for Weechat that lets Weechat communicate over the Matrix protocol.
+
+# Project Status
+
+Weechat-Matrix already supports large parts of the Matrix protocol.  See our [TODO](https://github.com/poljar/weechat-matrix/blob/master/TODO.md) list for more details.  Encryption support is still in the works.
+
+# Installation
+
+Installation is easy.  As your regular user, just run: `make install` in this repository directory.
+
+The following python modules must also be available on your system:
+
+- webcolors
+- http\_parser
+
+Note that weechat only supports Python2 OR Python3, and that setting is determined at the time that Weechat is compiled.  Weechat-matrix works with Python2 and Python3, but you will have to install the python dependencies accordingly.  Python2 is a good first guess.
+
+When encryption is ready, the plugin will also depend on the [OLM encryption library](https://git.matrix.org/git/olm/) being properly installed on the system.
```

Comment:
The OLM c library is a implicit dependency of the python bindings. It probably makes more sense to mention the bindings instead of the underlying lib: https://github.com/poljar/python-olm

Reply:
Sure.  Out of curiosity, what was the motivation to fork the python bindings?